app/controllers/my_module_comments_controller.rb

```diff
@@ -153,7 +155,9 @@ def destroy
               module: @my_module.name
             )
           )
-          render json: {}, status: :ok
+          render json: { linked_id: @my_module.id,
```

Comment:
Same here, add comment `# Used for counter badge` or something along those lines.